src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
-public class MyHashMap<K, V> implements MyMap<K, V> {
+import java.util.Objects;
+
+public class MyHashMap<K,V> implements MyMap<K,V> {
+    private final int defaultCapacity = 16;
+    private final float loadFactor = 0.75f;
```

Comment:
```suggestion
    private staticfinal int DEFAULT_CAPACITY = 16;
    private static final float LOAD_FACTOR = 0.75f;
```
константи це `static final` змінні, і вони через CAPS_CASE зазвичай пишуться (якщо це примітиви або immutable класи)